layouts/partials/alert.html

```diff
@@ -1,10 +1,16 @@
 <div class="mx-auto mb-12">
+    <div class="inline-block relative overflow-visible">
+        <!-- Relatively positioned mascot. Remove or comment this div to run an alert without it. -->
+        <div class="absolute top-0 left-0" style="left: -10rem; top: -2rem">
+            <img class="w-48" src="/images/mascot/pulumipus.svg" alt="Pulumi mascot">
+        </div>
         <a href="{{ relref . "/webinars/pulumi-update-2020-04-29" }}"
-           class="block p-2 bg-purple-400 hover:bg-purple-300 transition-all items-center text-purple-100 leading-none rounded-full flex md:inline-flex">
-            <span class="flex rounded-full bg-orange-600 uppercase text-white px-2 py-1 text-xs font-bold mr-2">New</span>
+            class="block p-2 bg-purple-400 hover:bg-purple-300 transition-all items-center text-purple-100 leading-none rounded-full flex md:inline-flex">
+            <span class="flex rounded-full bg-orange-700 border-2 border-orange-600 uppercase text-white px-2 py-1 text-xs font-bold mr-2">New</span>
             <span class="text-left flex-auto text-white text-sm md:text-base leading-normal">
-                Join us on April 29 for a fun walk-through of the latest Pulumi features and capabilities with Corey Quinn.
+                Join us on April 29 for a fun walk-through of the latest Pulumi features with Corey Quinn.
```

Comment:
```suggestion
                Pulumi 2.0 is available! Learn more here.
```

The CTA here should be to learn more about and use the product -- if they join the webinar, great, but that too's purpose is to drive people to learn more about and use the product.

Reply:
Ah, also the link should go to /superpowers. It seem to go to the webinar page right now.